pkg/commands/access/actions/list.go

```diff
@@ -39,5 +54,49 @@ func AccessList() *cobra.Command {
 		},
 	}
 
+	flags := cmd.Flags()
+	flags.StringVarP(&integrationFilter, integrationFilterFlagName, "i", "", "The integration id or type/name, for example: \"aws-account/My AWS integration\"")
```

Comment:
the type the user should use it the type id? is it exposed to him anywhere? ususally we show the display name to the user

Reply:
its the type ID and it exposed in the `apono inventory integrations` command

Reply:
ok